rcsb/db/mongo/PdbxLoader.py

```diff
@@ -783,8 +787,18 @@ def loadWorker(self, dataList, procName, optionsD, workingDir):
                     cId = cL[0].getName() if useNameFlag else cL[0].getProp("uid")
                     cIdD[cId] = locatorObj
                     containerList.extend(cL)
-            # -- Apply methods to each container -
+            #
             for container in containerList:
+                # -- Assign temporary attribute with internal identifier for computed models
+                if databaseName in ["pdbx_comp_model_core"]:
+                    eObj = container.getObj("entry")
+                    sourceId = eObj.getValue("id", 0)
+                    compModelInternalId = compModelIdMapD.get(sourceId, sourceId)
```

Comment:
Is the conversion to internalId handled by the RepositoryProvider?

Reply:
Yes, I created a new method here: https://github.com/rcsb/py-rcsb_utils_repository/blob/992480bfdefa0ad18f57bbb947a6038dc7468de2/rcsb/utils/repository/RepositoryProvider.py#L1243
I had asked Li to review that PR earlier, and just a moment ago I realized I had to make a very small bug fix so just pushed that too.